h2o-docs/src/product/data-sources.rst

```diff
@@ -105,4 +105,52 @@ These are available in the Bluemix/Softlayer management console.
       </property>
     </configuration>
 
+Google Cloud Storage Connector for Hadoop & Spark
+~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I get an error when I build locally because the heading underline is too short. It's just a warning, so I don't think it will cause a build failure.
Google Cloud Storage Connector for Hadoop & Spark
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~